services/graphql/src/services/reporting.js

```diff
@@ -0,0 +1,64 @@
+const Event = require('../mongoose/models/event');
+
+const { isArray } = Array;
+
+const aggregate = async ({ criteria, start, end }) => {
+  const date = { $gte: start, $lte: end };
```

Comment:
The start and end dates should be set outside of this method... that way, `aggregate` takes one argument... the `$match` value. Makes it easier to understand what's happening if the criteria is built in one place instead of two.